brick2221/reduction/PipelineRerunNIRCAM-LONG.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python
+import os
+# do this before importing webb
+os.environ["CRDS_PATH"] = "/orange/adamginsburg/jwst/w51/crds/"
```

Comment:
we should have one location on hipergator for this... probably should just drop w51 and brick...